samples/drivers/i2s/i2s_codec/Kconfig

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+source "Kconfig.zephyr"
+
+config I2S_INIT_BUFFERS
+	int "number of i2s transfer buffer"
+	default 2
+	help
+	  i2s init buffers used in transfer
+
+config SAMPLE_FREQ
+	int "sample rate"
+	default 48000
+	help
+	  sample rate
```

Comment:
these help texts are not sufficient, remember this will show in the menuconfig ui for users, it should have proper capitalization and grammar too